google/cloud/internal/curl_wrappers.cc

```diff
@@ -168,6 +169,17 @@ class CurlInitializer {
   CurlInitializer() { curl_global_init(CURL_GLOBAL_ALL); }
   ~CurlInitializer() { curl_global_cleanup(); }
 };
+
+auto constexpr kMaxDebugLength = 128;
```

Comment:
Is it better to make this explicitly `std::size_t` (and then let `std::min` deduce it)?